src/monodroid/jni/monodroid-glue.cc

```diff
@@ -2122,9 +2150,12 @@ JNICALL Java_mono_android_Runtime_register (JNIEnv *env, jclass klass, jstring m
 	char *type;
 	const char *mt_ptr;
 	MonoDomain *domain = monoFunctions.domain_get ();
+	timing_period total_time;
 
-	long long start_time = current_time_millis (), end_time;
-	log_info (LOG_TIMING, "Runtime.register: start time: %lli ms\n", start_time);
+	if (XA_UNLIKELY (utils.should_log (LOG_TIMING))) {
+		total_time.mark_start ();
+		log_info (LOG_TIMING, "Runtime.register: start");
```

Comment:
Is there a particular reason why the "start" messages no longer contain timestamps?  Are we assuming that the `adb logcat` times are sufficient, or that the "start" message doesn't *need* a time, and we only care about *elapsed* times?

If we only care about end+elapsed times, why log the start at all?

Reply:
Those messages have no useful meaning, I forgot to remove this. The timestamps were not very useful because of logcat having them as well and also they were just a large number of milliseconds - without information about the timezone etc. With the change here, introducing new timing source, they make even less sense - the monotonic clock measures time from "something" (usually boot, but that's not guaranteed) which gives us no information whatsoever. We care only about elapsed time. I'll remove this log call.